src/contexts/svg.ts

```diff
@@ -171,14 +171,14 @@ export class SvgContext implements ITypesetterContext<SVGElement> {
     // if element has a text element descendent, select it and return it
     const selected = element.querySelector("text");
     if (selected != null) {
-      let parentElement = element.parentElement;
+      let parentElement = selected.parentElement;
```

Comment:
notably, it was a bug that we were not previously using the selected text element.